src/engraving/rendering/dev/horizontalspacing.h

```diff
@@ -22,6 +22,9 @@
 #ifndef MU_ENGRAVING_HORIZONTALSPACINGUTILS_DEV_H
 #define MU_ENGRAVING_HORIZONTALSPACINGUTILS_DEV_H
 
+#include <dom/notedot.h>
+
+

```

Comment:
This change looks unnecessary / unintended

Reply:
oops, removed